bin/genesis

```diff
@@ -90,6 +133,34 @@ setup() {
 		exit 3
 	fi
 
+	if [[ -f ${DEPLOYMENT_ROOT}/.genesis_deps ]]; then
+		errors=""
+		while IFS='' read -r line || [[ -n "$line" ]]; do
+			cmd=$(echo $line | cut -d ":" -f1)
+			version=$(echo $line | cut -d ":" -f2- | auto_sed 's/^ +//' | auto_sed 's/ +$//')
```

Comment:
Withdrawn.  However, can we go ahead and use `perl -pi -e` (as below, line 146) instead of sed, to be more uniform?